pkg/cmd/kafka/create/create.go

```diff
@@ -363,12 +361,12 @@ func runCreate(opts *options) error {
 
 // set type to store the answers from the prompt with defaults
 type promptAnswers struct {
-	Name            string
-	Size            string
-	Region          string
-	CloudProvider   string
-	AccountID       string
-	CloudProviderID string
+	Name          string
+	Size          string
+	Region        string
+	CloudProvider string
+	MarketplaceID string
```

Comment:
marketplaceID would be `aws` `rhm` etc.  Value that we passing represent "Cloud Provider Account ID" that have been used to purchase kafka.

Reply:
Referring to the KFM OpenAPI. It seems that would be [marketplace](https://github.com/redhat-developer/app-services-sdk-go/blob/7fc1e6307584087b8fab7706f1947550e1ef71a0/.openapi/kas-fleet-manager.yaml#L1095).
Would like the name of this flag to convey it is an `accountID`, while at the same time it should have something to indicate it is related to `marketplace`. Renaming it to `MarketplaceAccountID`

Reply:
FYI @pb82